install.sh

```diff
@@ -521,8 +521,8 @@ sudo pip install -q -U tqdm
 # The BlueView SDK for the Teledyne imaging sonar
 if [[ ! -z "$BVTSDK_PASSWORD" ]]; then
 	instlog "Decrypting and installing the BlueView SDK"
-	cd $CATKIN_DIR/src
-	curl -s https://raw.githubusercontent.com/uf-mil/installer/master/bvtsdk.tar.gz.enc | \
+	cd $CATKIN_DIR/src/mil_common/drivers/mil_blueview_driver/
```

Comment:
No trailing slash, here. The use of `cd` implies that it is a directory and it is not convention for the file.